chat-widget/src/components/livechatwidget/common/authHelper.ts

```diff
@@ -33,7 +33,16 @@ const handleAuthentication = async (chatSDK: OmnichannelChatSDK, chatConfig: Cha
             TelemetryHelper.logActionEvent(LogLevel.ERROR, { Event: TelemetryEvent.ReceivedNullOrEmptyToken });
             throw new Error(WidgetLoadCustomErrorString.AuthenticationFailedErrorString);
         }
+    } else if (chatSDK && chatSDK.chatSDKConfig?.getAuthToken) {
```

Comment:
It could be simplified to `chatSDK?.chatSDKConfig?.getAuthToken` instead since it's `TS`